### ISSUE TITLE
:wheelchair: a11y(social share): Link list not accessible

### ISSUE THREAD

**Issue by carolabes** [baloise/design-system#1420]
### Description of this issue

* Link list not accessible.

<img width="1594" alt="seite-teilen" src="https://github.com/baloise/design-system/assets/172396366/03ad959e-3cbe-4e31-8363-f1dc6bf1d575">

### How to fix it

* Use aria-controls and aria-expanded

### Additional Information

Level A
https://www.w3.org/WAI/WCAG21/Understanding/keyboard.html

aria-controls: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-controls
aria-expanded: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-expanded

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by yannickholzenkamp** [comment]
Hi @carolabes 

Thanks for filing this issue. I just checked it, but I do not think a fix in the Design-System is necessary.
The social share "Seite teilen" button is not part of the Design-System footer, but just a regular `bal-button`. 
The necessary aria attributes can be passed to the `bal-button` using the `aria` property (see https://design.baloise.dev/?path=/docs/components-containment-button--documentation).

Is this okay for you or do you have an issue with the `aria` property of the `bal-button`?

I would close this issue. In case it is not resolved for your usecase, please let us know or re-open the ticket.
